lib/resources/dstu2/condition.yaml

```diff
@@ -139,12 +139,12 @@ fields:
     }
   binding:
     description: A category assigned to the condition.
-    note: Currently `diagnosis` and `problem` category codes are supported. Code `diagnosis` is bound to condition category codes and `problem` is bound to argonaut extenstion codes terminology system shown below.
+    note: Category codes `diagnosis`, `problem` and `health-concern` are currently supported for search and retrieve functionality. Code `health-concern` is currently not supported for conditions add and update functionality.
     terminology:
     - display: Condition Category Codes
       system: http://hl7.org/fhir/condition-category
-    - display: Argonaut Extension Code
-      system: http://argonaut.hl7.org/extension-codes
+    - display: Argonaut Valueset Codes
+      system: http://argonautwiki.hl7.org/index.php?title=Problems_and_Health_Concerns#problems_and_health_concerns_code_extensions
```

Comment:
Is this the current link we should be using. Drew has always been pointing me to pages at http://build.fhir.org/ig/Healthedata1/Argo-DSTU2/ when referencing Argonaut implementation guides